token.go

```diff
@@ -34,6 +34,13 @@ type Token struct {
 	Claims    Claims                 // Claims is the second segment of the token in decoded form
 	Signature []byte                 // Signature is the third segment of the token in decoded form.  Populated when you Parse a token
 	Valid     bool                   // Valid specifies if the token is valid.  Populated when you Parse/Verify a token
+
+	encoders
+}
+
+type encoders struct {
```

Comment:
its not private, but still we probably could use some comments here, especially for this outer struct